### DISCUSSION TITLE
simple nmc simulation giving odd results

### DISCUSSION THREAD

**Discussion by skywo1f** [pybamm-team/PyBaMM#2460] [General]
The code is 
`import pybamm

chemistry = pybamm.parameter_sets.Mohtat2020
parameter_values = pybamm.ParameterValues(chemistry=chemistry)
model = pybamm.lithium_ion.DFN()

sim = pybamm.Simulation(model,parameter_values=parameter_values)
sim.solve([0, 3600])  # solve for 1 hour
sim.plot()`

For DFN, It does not converge:
`At t = 3.73014e-05 and h = 1.14201e-19, the corrector convergence failed repeatedly or with |h| = hmin.
At t = 3.73008e-05 and h = 5.76126e-21, the corrector convergence failed repeatedly or with |h| = hmin.
At t = 3.73009e-05 and h = 3.12415e-21, the corrector convergence failed repeatedly or with |h| = hmin.
At t = 3.73007e-05 and h = 9.11632e-20, the corrector convergence failed repeatedly or with |h| = hmin.`

for SPM, it converges, but the voltage goes to -7e36V

**Comment by rtimms** [comment]
The Mohtat2020 initial conditions correspond to a fully discharged state. On the latest version you get the error `SolverError: Events ['Minimum voltage'] are non-positive at initial conditions` if you try to discharge. 

You can do e.g. a 1C charge as follows
```python3
import pybamm

parameter_values = pybamm.ParameterValues("Mohtat2020")
parameter_values["Current function [A]"] = -5
model = pybamm.lithium_ion.SPM()
sim = pybamm.Simulation(model, parameter_values=parameter_values)
sim.solve([0, 3600])  # solve for 1 hour
sim.plot()
```

**Comment by skywo1f** [comment]
I changed the voltage cutoffs to the more sensible range (3.1 to 4.1V). but now I am getting the " Events ['Minimum voltage'] are non-positive at initial conditions" error.

**Reply by valentinsulzer** [reply]
The initial conditions in the parameter set give a voltage below 3.1 V. The easiest workaround for this is to do `sim.solve([0,3600], initial_soc=1)`. See https://github.com/pybamm-team/PyBaMM/blob/develop/examples/notebooks/models/electrode-state-of-health.ipynb for more details

**Reply by skywo1f** [reply]
thanks, that also solved a separate issue I was having where different parameter values had different starting conditions.
